.eslintrc.js

```diff
@@ -8,18 +8,20 @@ const rulesDirPlugin = require('eslint-plugin-rulesdir');
 
 const { baseConfig } = eslintConfig(false);
 
-rulesDirPlugin.RULES_DIR = path.resolve(path.join(
-  __dirname,
-  'node_modules',
-  'gatsby',
-  'dist',
-  'utils',
-  'eslint-rules',
-));
+rulesDirPlugin.RULES_DIR = path.resolve(
+  path.join(
+    __dirname,
+    'node_modules',
+    'gatsby',
+    'dist',
+    'utils',
+    'eslint-rules',
+  ),
+);
```

Comment:
If we don't prefer the former, then there should probably be a linter rule for that.

Reply:
Can you elaborate a bit more? I'm not sure what are you referring to.

Reply:
If this code needs reformatting, then there should be a linter rule telling us that.

Reply:
Previously, I didn't know eslint-plugin-prettier would make ESLint tell us if somethings wrong.